server/routes/service.js

```diff
@@ -1,3 +1,153 @@
-const transport = require('../routes/transport');
-const db = require('../db/db'); 
+require('dotenv').config();
+const bcrypt = require('bcrypt');
+const saltRounds = 10;
+const jwt = require('jsonwebtoken');
+const db = require('../db/db');
 
+
+// Check if the user is already authenticated
+function checkAuthStatus(authCookie) {
+  jwt.verify(authCookie, process.env.ACCESS_TOKEN_SECRET, (err) => {
+    if (err) {
+      console.log(err);
+      return '403';
+    }
+  });
+};
+
+// Check user existense
+async function signInDbQuery(name, pw) {
+  let checkedReturn;
+  const data = await db.checkIn(name);
+  if (data) {
+    const id = data.user_id;
+    const uName = data.user_name;
+    const err = data;
+    if (id && uName) {
+      await bcrypt.compare(pw, data.password)
+        .then(async (result) => {
+          const authenticatedUser = { id, name: uName };
+          if (result) {
+            checkedReturn = setToken(authenticatedUser);
+          } else {
+            checkedReturn = '401'
+          }
+        });
+    } else {
+      console.error(err);
+    }
+  } else {
+    checkedReturn = '404'
+  }
+  return checkedReturn;
+};
+
+// Create a new user
+async function signUpDbQuery(name, pw) {
+  let createdReturn;
+  const hashedPassword = await bcrypt.hash(pw, saltRounds);
+  const data = await db.checkUp(name);
+  if (data) {
+    const uName = data.user_name;
+    const err = data;
+    if (uName) {
+      createdReturn = '401'
+    } else {
+      console.error(err);
+    }
+  } else {
+    const dataC = await db.createUser(name, hashedPassword);
+    console.log('1 new user added');
+    const authenticatedUser = { id: dataC.user_id, name: dataC.user_name };
+    if (dataC.user_id) {
+      createdReturn = setToken(authenticatedUser);
+    } else {
+      createdReturn = '500'
+    }
+  }
+  return createdReturn;
+}
+
+function setToken(authenticatedUser) {
+  const accessToken = jwt.sign(authenticatedUser, process.env.ACCESS_TOKEN_SECRET);
+  return accessToken;
+}
+
+// Middlewear for sending cookie information to client side
+function tokenCheck(authCookie) {
+  let checkedToken;
+  if (authCookie === null) {
+    checkedToken = '401';
+  } else {
+    jwt.verify(authCookie, process.env.ACCESS_TOKEN_SECRET, (err, user) => { 
+      if(err) {
+        checkedToken = '403';
+      } else {
+        checkedToken = user;
+      }
+    });
+  }
+  return checkedToken;
+}
+
+// Get user's lists on the sign in
+async function queryList(usrId) {
+  let queryResult
```

Comment:
This doesn't need to be defined here. You can just return lists or 500 on lines 98 and 100 respectively. This will also let you remove that else statement.